provider/apiutils/apiutils.go

```diff
@@ -109,6 +110,14 @@ func GetClassLabel(o api.Object) (string, bool) {
 	return class, found
 }
 
+func SetMachineNamespaceLabel(o api.Object, namespace string) {
```

Comment:
do you need this functions? It will save only one argument from 3. I don't see big benefits here or extending of SetLabel logic

Reply:
I am trying to follow the existing conversion with `SetClassLabel` and `SetManagerLabel` being already there.